libobs-opengl/gl-shader.c

```diff
@@ -220,8 +220,20 @@ static bool gl_shader_init(struct gs_shader *shader,
 	if (!gl_success("glGetShaderiv"))
 		return false;
 
-	if (!compiled)
+	if (!compiled) {
+		GLint infoLength = 0;
+		glGetShaderiv(shader->obj, GL_INFO_LOG_LENGTH, &infoLength);
+
+		char* infoLog = malloc(sizeof(char) * infoLength);
```

Comment:
Style:  Place asterisk (*) next to variable name instead of next to type.

```suggestion
		char *infoLog = malloc(sizeof(char) * infoLength);
```